src/api/cli/cli.go

```diff
@@ -305,3 +309,72 @@ func printJSON(obj interface{}) error {
 
 	return nil
 }
+
+// readPasswordFromTerminal promotes user to enter password and read it.
+func readPasswordFromTerminal() ([]byte, error) {
+	// Promotes to enter the wallet password
+	fmt.Fprint(os.Stdout, "enter password:")
+	bp, err := terminal.ReadPassword(0)
```

Comment:
should we use os.Stdin?

Reply:
`os.Stdin` is type of `*os.File`, while the `ReadPassword` accepts `int` type as argument. I tried to use `syscall.Stdin`, but it has compilation problem, would have error: 

```
cannot use syscall.Stdin (type syscall.Handle) as type int in argument to terminal.ReadPassword
```

Reply:
Nevermind, its safe to use 0, syscall.Stdin is always 0 anyway

Reply:
Yes, syscall.Stdin is 0, but not for windows, as the `build` flag in `syscall_unix.go` is
`// +build darwin dragonfly freebsd linux netbsd openbsd solaris`.

Reply:
What is it in windows? Then do we need to do `int(syscall.Stdin)`?

Reply:
I see, `syscall_windows.go:	Stdin  = getStdHandle(STD_INPUT_HANDLE)`

then we probably need to do `int(os.Stdin.Fd())` or `int(syscall.Stdin)`

Reply:
I will test `int(syscall.Stdin)` on windows to see if it is able to work

Reply:
OK,` int(syscall.Stdin)` works, I will use it.